polling_stations/apps/file_uploads/templates/file_uploads/index.html

```diff
@@ -0,0 +1,178 @@
+{% extends "base.html" %}
+
+{% load i18n %}
+{% load static %}
+
+{% block extra_page_css %}
+<style>
+  button:disabled {
+    background-color: lightgrey;
+    border-color: darkgrey;
+  }
+</style>
+{% endblock extra_page_css %}
+
+{% block content %}
+<div class="card">
+
+    <noscript>
+      <div class="card error-summary">
+          <h2>Javascript disabled</h2>
+          <p>To upload files, please enable javascript.</p>
+      </div>
+    </noscript>
```

Comment:
Given this is for a tiny set of users that we know, then feel free to ignore this comment, but given what you say about specific JS requirements, it might be worth adding some sort of feature check in future that tells the user they are using an unsupported browser, rather than just a boolean JS enabled/disabled.

Also, this feels like something that could live in the base theme as a JS utility function.